README.md

```diff
@@ -100,8 +100,52 @@ First the entry, that is filtered by URL.
 Second the stream, that is filtered by the `filter` field.
 Finally some cases have a rover_bbox or rover_countries field (when there are several CRSs for the same mountpoint).
 
+#### entries
+The json file is composed mainly by a list of entries.
+Each entry is identified by a `name` and `urls`.
+The name is for informative reasons.
+The urls is a list of fully defined URLs, including protocol (`http` or `https`), domain and port.
+Each URL (including port) must appear only once in the full catalog.
+In case a service is available under several URLs (for instance, IP and hostname), all should be added in the same entry.
+The `reference` section must contain the source where this information is published.
```

Comment:
Minor: we're open to domain squatting here, if the domain changes ownership someone could publish malicious content that we would then link to.